keycloak/keycloak-admin-ui
# Partial import phase 2

## Brief Description
This is phase 2 of the Partial import implementation.  Phase 3 will be the final phase where the imported data is sent to the Keycloak server.

Partial import modal now handles the imported JSON and presents the user with import options as per [Marvel](https://marvelapp.com/prototype/fg26f9b/screen/75624791).  

Note: The design did not include functionality for handling JSON files with multiple realms.

## Verification Steps
1. Go to `Realm settings`.
2. Select Partial import form the Action dropdown.
3. Import a previously exported Keycloak json. (Samples included in test data for this PR. `kcexport.json` is a good one to look at)
4. Verify that modal presents the user with options as shown in the design, plus multi-realm functionality.

## Checklist:

- [ ] Help has been implemented (not applicable)
